packages/test/package.json

```diff
@@ -3,24 +3,40 @@
   "version": "3.0.1",
   "description": "Testing utilities for Rest Hooks",
   "sideEffects": false,
-  "main": "dist/index.cjs.js",
-  "module": "lib/index.js",
+  "main": "dist/index.cjs",
```

Comment:
for actual back compat support, you should NOT use `type: module`, and CJS files should be `.js`, and ESM files should only be `.mjs`.

(Since your engines is `>= 10`, you're still supporting many pre-exports, pre-ESM node versions)

Reply:
I didn't seem to see a lot of adoption of mjs extension? Is that what node cares about? Does this include the files in the 'exports' part?

Reply:
Correct me if I'm wrong, but older versions of node don't support 'exports' at all - and thus will read from 'main'?

Reply:
As I understand webpack 4 uses module as it does not support exports. So removing it would change its behavior. Why would removing it help anything?

Reply:
I'm not saying to remove the `module` field, i'm saying to remove `"type": "module"` - which has only one purpose, to change how `.js` is parsed. By default, and properly, it's CJS, but with type module it's treated as ESM.

node always considers `.mjs` to be ESM, because that's the proper extension for it.

Reply:
ah, I get it now. I got confused because of which line this message was bound to. Will remove type: module - i didn't fully understand its purpose before

Reply:
I found https://nodejs.org/docs/latest-v15.x/api/packages.html#packages_conditional_exports and it seems to suggest that "type": "module" works well with import having a esm